Makefile

```diff
@@ -36,8 +36,7 @@ install:
 .PHONY: install
 
 install-dependencies:
-	go mod vendor
```

Comment:
Sorry, I am not sure why `go mod vendor` is used here. But it's useless without either `-mod=vendor` flag for all go commands or `GOFLAGS=-mod=vendor` env variable.